backend/deepchecks_monitoring/ee/api/v1/slack.py

```diff
@@ -130,7 +130,8 @@ async def installation_callback(
         )
 
     try:
-        installation = utils.finish_installation(code)
+        redirect_path = str(request.url_for('slack-installation-callback'))
```

Comment:
is it safe to use `url_for` even if the server is behind a proxy or something?

Reply:
we also use it in eula redirect and authorization redirect, I wonder if we need to replace it in all.
we do already have the `deployment_url` in the settings

Reply:
I checked with local proxy, it works if the proxy adds a "preserver host" header

Reply:
cool, thanks!